dockers/docker-orchagent/swssconfig.sh

```diff
@@ -46,7 +46,7 @@ elif [ "$HWSKU" == "Force10-S6000-Q32" ]; then
 elif [ "$HWSKU" == "Arista-7050-QX32" ]; then
     SWSSCONFIG_ARGS+="td2.32ports.buffers.json td2.32ports.qos.json "
 elif [[ "$HWSKU" == "ACS-MSN27"* ]]; then
-    sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/msn27xx.32ports.buffers.json.j2 > /etc/swss/config.d/msn27xx.32ports.buffers.json
+    sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/msn27xx.32ports.buffers.json.j2 > /etc/swss/config.d/msn27xx.32ports.buffers.json || exit 1
```

Comment:
what about `set -e` for this script?

Reply:
The swssconfig.sh could be improved in future. Currently there is no assumption that every command will success there, except the obvious line such as sonic-cfggen.

Reply:
I didn't get why there's no assumption that every command will success? right now there're sonic-cfggen and swssconfig commands. I think both of them need to be successful after execution.